### PR TITLE
kie-issues#108: Upgrade patternfly react modules to latest on kie-tools

### PR DESCRIPTION
**Pull request by pefernan**
Upgrades patternfly to the latest [release](https://www.patternfly.org/2022.16/developer-resources/release-notes).

### CODE REVIEW

**Comment by pefernan** [comment]
@paulovmr @jesuino @caponetto @AjayJagan @Sara4994 @ljmotta @tiagobento guys, I tried everything I was aware of to make sure I broke nothing, but I'd like that you please review your components/webapps to identify any possible issue and let me know.

Thanks!

File: packages/uniforms-patternfly/src/__tests__/DateField.test.tsx [hunk 1]
```diff
@@ -108,15 +108,28 @@ test("<DateField> - renders a input with correct value (specified)", () => {
 });
 
 test("<DateField> - renders a input which correctly reacts on change (DatePicker)", () => {
+  // Skipping due to a known timezone issue.
```
Comment (ljmotta) [inline packages/uniforms-patternfly/src/__tests__/DateField.test.tsx]: I guess you forgot this comment here.

**Review by ljmotta — COMMENTED** [review]
Thanks for your PR @pefernan. I've made just one comment inline.

**Review by jstastny-cz — APPROVED** [review]
Looks good.
I've reviewed KIE sandbox and didn't notice any discrepancies.

**Review by tiagobento — APPROVED** [review]

File: packages/online-editor/src/editor/DmnRunner/DmnRunnerDockToggle.tsx [hunk 2]
```diff
@@ -28,6 +28,7 @@ interface Props {
 }
 
 export function DmnRunnerDockToggle(props: Props) {
+  console.log("dmnrunner toggle!");
```
Comment (caponetto) [inline packages/online-editor/src/editor/DmnRunner/DmnRunnerDockToggle.tsx]: Leftover?

**Review by caponetto — APPROVED** [review]
Thanks, @pefernan!

**Review by jesuino — APPROVED** [review]

**Review by paulovmr — APPROVED** [review]

**Review by AjayJagan — APPROVED** [review]
Thanks @pefernan

File: packages/online-editor/src/editor/DmnRunner/DmnRunnerDockToggle.tsx (hunk 2 above)
Comment (pefernan) [reply packages/online-editor/src/editor/DmnRunner/DmnRunnerDockToggle.tsx]: @caponetto touché.. cleaning it now

**Pull request merged**